content/en/docs/concepts/security/security-checklist.md

```diff
@@ -206,6 +206,14 @@ SELinux is only available on Linux nodes, and enabled in
 [some Linux distributions](https://en.wikipedia.org/wiki/Security-Enhanced_Linux#Implementations).
 {{< /note >}}
 
+## Logs and auditing
+
+- [ ] Audit logs, if enabled, are protected from general access
+- [ ] The `/logs` API is disabled (kube-apiserver runs with `--enable-logs-handler=false`)
```

Comment:
This may be a nit, but I quite like the phrasing on the message that it is disabled by default and you'll need to set the flag to true to enable it. I think it sounds more idiomatic and clarifies the point better.

Reply:
```suggestion
- [ ] The `/logs` API is disabled (you are running kube-apiserver with
  `--enable-logs-handler=false` and not the default setting where it is enabled).
```

Reply:
How's this?

Reply:
SGTM, thanks!